charts/astronomer/templates/_helpers.yaml

```diff
@@ -237,8 +237,10 @@ s3:
   value: {{ .Release.Name }}-registry
 - name: REGISTRY__PORT
   value: {{ .Values.ports.registryHTTP | quote }}
-- name: NATS__URL
-  value: nats://{{ .Release.Name }}-nats:4222
+- name: NATS__SERVERS
+  value: {{ template "houston.nats.servers" . }}
+- name: NATS__CLUSTER_ID
+  value: {{ .Release.Name }}-stan
```

Comment:
@andriisoldatenko this is where we are getting logs for `stan.clusterID` value from in Houston API.  I don't have a way with the NATS library to receive the clusterID for NATS too.  We're only getting the URLs that we are using for NATS from the `servers` reference OR from the NATS auto discovery (what you were demoing yesterday with the automatic update of the server urls).  So it seems like STAN is giving us the NATS pod urls and then we are able to make that NATS client connection from the library directly to NATS Server.

Reply:
This looks like it'll resolve correctly with the changes were recently merged within Houston that changes the schema to JSON.

Reply:
This isn't a requested change.  I'm wondering if we should change what we're logging in Houston API/Houston Worker and if it's important to show the cluster ID in the logs like they are currently implemented:

Example (note: `astrodev-stan`):
```
2020-08-24T18:27:05 INFO Reconnected to cluster astrodev-stan using client houston-deployment-deleted-worker-66dc5f1b-5ca4-4cb6-9513-fc4776e63e8a and url nats://astrodev-nats:4222
```

Reply:
Removed the `clusterID` here https://github.com/astronomer/houston-api/pull/438